From khali@linux-fr.org Sun Dec 18 08:15:45 2005
Date: Sun, 18 Dec 2005 17:18:01 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: Martin Hicks <mort@bork.org>
Subject: [PATCH 09/11] i2c: i2c-ibm_iic add I2C_CLASS_HWMON
Message-Id: <20051218171801.1b0c3fd7.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-ibm_iic-hwmon-class.patch

From: Martin Hicks <mort@bork.org>

Add the ibm_iic driver to the HWMON class so it will scan the
bus for connected hardware monitor sensors.

Signed-off-by: Martin Hicks <mort@bork.org>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/i2c/busses/i2c-ibm_iic.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/i2c/busses/i2c-ibm_iic.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-ibm_iic.c
@@ -725,6 +725,7 @@ static int __devinit iic_probe(struct oc
 	strcpy(adap->name, "IBM IIC");
 	i2c_set_adapdata(adap, dev);
 	adap->id = I2C_HW_OCP;
+	adap->class = I2C_CLASS_HWMON;
 	adap->algo = &iic_algo;
 	adap->client_register = NULL;
 	adap->client_unregister = NULL;
